jekyll/_cci2/rotate-project-ssh-keys.adoc

```diff
@@ -0,0 +1,110 @@
+
+[#rotate-a-user-key-github]
+=== Rotate a user key
+
+If you have set up user keys for your project, follow these steps:
+
+. Take note of the current key information to rotate, including the fingerprint. You can also click the keyname to open the related Github page that should list the public key.
+. Delete the deploy key by clicking the **X**.
+. Add a new user key by clicking **Add User Key**.
+. Go to GitHub’s User settings to delete the matching public key. The github URL is typically `https://github.com/settings/keys`or you may already have the page open if you clicked on the keyname in step 1. Removing any key titled `CircleCI` created before this rotation is recommended. The new public SSH key will be automatically added once the old key is deleted (page may need to be refreshed to view the key).
+
+CAUTION: The user key name contains the project name, however, a user key may give access to more projects within the organization.
+
+NOTE: If using organization SSO, you can take note of which one is currently authorized. For the newly created key, if using organization SSO, and access is needed, you will need to reauthorize it.
+
+[#rotate-an-additional-SSH-key-github]
+=== Rotate an additional SSH key
+
+If you are using additional SSH keys in your project, follow these steps:
+
+. Take note of the existing key to know which target system it is used for. This could be a VCS, a machine or other SSH based system.
+. Delete the deploy key by clicking the **X**.
+. Remove the authorization to use that key (the public key) from the target system. Typically, the public key will be authorized in the target system, and the configuration varies depending on the target system.
+. Recreate the key and the authorization in the target system.
+
+[#bitbucket-projects]
+== Bitbucket projects
+
+Go to **Project Settings > SSH Keys** to view SSH keys set up for your project.
+
+[#rotate-a-deploy-key-bitbucket]
+=== Rotate a deploy key
+
+. Take note of the current key information to rotate.
+. Delete the deploy key by clicking the **X**.
+. Add a new deploy key by clicking **Add Deploy Key**.
+. Go to Bitbucket’s project admin access-key settings to delete the matching public key. The Bitbucket URL is typically `https://bitbucket.org/<org>/<project>/admin/access-keys/`. The keys are named `CircleCI`. Since the fingerprint hash is not shown, it is recommended to remove any key titled `CircleCI` created before the rotation. The new public SSH key will be automatically added once the old key is deleted (page may need to be refreshed to view the key).
+
+
+[#rotate-a-user-key-bitbucket]
+=== Rotate a user key
+
+. Take note of the current key information to rotate.
+. Delete the deploy key by clicking the **X**.
```

Comment:
```suggestion
. Delete the user key by clicking the **X**.
```

Reply:
yes I just realized that too, all three should be followed by
Add the private key by clicking Add SSH Key. It is recommended you name the key [gitlab.com](http://gitlab.com/). Paste in the private key you created in the step above and click Add SSH Key again to finalize.